nifi-nar-bundles/nifi-extension-utils/nifi-hadoop-utils/src/main/java/org/apache/nifi/processors/hadoop/AbstractHadoopProcessor.java

```diff
@@ -67,40 +61,14 @@
  */
 @RequiresInstanceClassLoading(cloneAncestorResources = true)
 public abstract class AbstractHadoopProcessor extends AbstractProcessor {
-    /**
-     * Compression Type Enum
-     */
-    public enum CompressionType {
-        NONE,
-        DEFAULT,
-        BZIP,
-        GZIP,
-        LZ4,
-        SNAPPY,
-        AUTOMATIC;
-
-        @Override
-        public String toString() {
-            switch (this) {
-                case NONE: return "NONE";
-                case DEFAULT: return DefaultCodec.class.getName();
-                case BZIP: return BZip2Codec.class.getName();
-                case GZIP: return GzipCodec.class.getName();
-                case LZ4: return Lz4Codec.class.getName();
-                case SNAPPY: return SnappyCodec.class.getName();
-                case AUTOMATIC: return "Automatically Detected";
-            }
-            return null;
-        }
-    }
 
     // properties
     public static final PropertyDescriptor HADOOP_CONFIGURATION_RESOURCES = new PropertyDescriptor.Builder()
             .name("Hadoop Configuration Resources")
             .description("A file or comma separated list of files which contains the Hadoop file system configuration. Without this, Hadoop "
                     + "will search the classpath for a 'core-site.xml' and 'hdfs-site.xml' file or will revert to a default configuration.")
             .required(false)
-            .addValidator(createMultipleFilesExistValidator())
+            .addValidator(HadoopValidators.MULTIPLE_FILE_EXISTS_VALIDATOR)
```

Comment:
Minor comment -- until I read the source code for this, my interpretation was that this validator ensured that *multiple files existed* -- i.e. one file provided would fail. Perhaps we can rename this `ONE_OR_MORE_FILES_EXIST_VALIDATOR`? Not a giant issue but potentially confusing.